libraries/eddystone-advertising-library/bower.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "eddystone-advertising-library",
+  "version": "0.3",
+  "homepage": "https://github.com/google/eddystone/tree/master/libraries/eddystone-advertising-library",
```

Comment:
Well the idea is that this is a cross platform library. Firefox OS has their own advertising API and Web Bluetooth will eventually have one as well. So anyone could write their own wrapper similar to [EddystoneChromeOS]() and add it to `platform.js` without having to worry about Eddystone's low level details.
@jyasskin who suggested this.